service-providers/dotmatrix/25-Q2-Marketing-Ecosystem-Report.md

```diff
@@ -0,0 +1,54 @@
+---
+Applicant: DotMatrix LLP.
+Type: Ecosystem & Community Growth
```

Comment:
To avoid ambitious with other parts of Ecosystem group tasks, I would suggest changing to more specific, Marketing and Adoption Growth.